platforms/android/demo/src/main/java/com/mapzen/tangram/android/MainActivity.java

```diff
@@ -88,6 +189,63 @@ public void onLowMemory() {
         view.onLowMemory();
     }
 
+    @Override public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+        final ArrayList<SceneUpdate> sceneUpdates = new ArrayList<>(1);
+        final String apiKey = "vector-tiles-tyHL4AY";
+        sceneUpdates.add(new SceneUpdate("global.sdk_mapzen_api_key", apiKey));
+        spinner.setSelection(7);
+        map.loadSceneFile(sceneString.getText().toString(),sceneUpdates);
```

Comment:
if you replace `onItemClick` with something like this:

```
int selectedIndex = 7; // custom
String sceneText = sceneString.getText().toString();
for (int i = 0; i < scenes.length; i++) {
    if (sceneText.equals(scenes[i])) {
        selectedIndex = i;
        break;
    }
}
spinner.setSelection(selectedIndex);
        
sceneString.clearFocus();
InputMethodManager in = (InputMethodManager) getSystemService(
Context.INPUT_METHOD_SERVICE);
in.hideSoftInputFromWindow(sceneString.getWindowToken(), 0);
```

it will add some polish by dismissing the keyboard on selection and detect the corresponding spinner item selected and update it to be the selected spinner item (right now if you click a style within the scenes list it wont update the spinner).